app/models/concerns/scripts/import.rb

```diff
@@ -75,8 +75,9 @@ def import_requires requires, scripts_data
       end
 
       def create_from_data data
-        parameters = data.delete('parameters')
-        requires   = require_attributes data.delete('requires')
+        parameters   = data.delete('parameters')
+        descriptions = data.delete('descriptions')
```

Comment:
```suggestion
        descriptions = data.delete('descriptions')
        parameters   = data.delete('parameters')
```
Solo para mantenerlos en orden alfabético =)

Reply:
genial